common/build/eslint-config-fluid/src/custom-rules/no-release-tags.js

```diff
@@ -0,0 +1,54 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+const { TSDocParser } = require("@microsoft/tsdoc");
+
+function hasReleaseTag(comment) {
+	const parser = new TSDocParser();
+	const parserContext = parser.parseString(comment);
+
+	const hasReleaseTag =
+		parserContext.docComment.modifierTagSet.isAlpha() ||
+		parserContext.docComment.modifierTagSet.isBeta() ||
+		parserContext.docComment.modifierTagSet.isPublic() ||
+		parserContext.docComment.modifierTagSet.isInternal();
+
+	return hasReleaseTag;
+}
+
+module.exports = {
+	meta: {
+		type: "problem",
+		docs: {
+			description: "This rule restricts any release tags on member class and interface.",
+			category: "Best Practices",
+		},
+		schema: [],
+		messages: {
+			releaseTagOnMember: "Release tag on member class / interface found.",
+		},
+	},
+	create(context) {
+		return {
+			ClassDeclaration(node) {
+				const sourceCode = context.sourceCode;
+				const comments = sourceCode.getCommentsInside(node);
+
+				// console.log("comments :", comments);
+
+				comments.forEach((comment) => {
+					// ESLint trims the asterisk of the comment while TSDocParser expects the original format of the comment block.
+					const formattedComment = `/** ${comment.value} */`;
+					if (hasReleaseTag(formattedComment)) {
+						context.report({
```

Comment:
This will result in us logging the same error once for each violating member of the class declaration. If we're only reporting 1 error (which I think is an okay place for us to start), we should probably only report 1 error, preferably with info about each of the violations.

We could implement this by saving off all of the violations as we iterate through the members, and then report once after the loop if we saved anything.